packages/cli/openapi-ir-to-fern/src/buildQueryParameter.ts

```diff
@@ -202,6 +246,50 @@ function getQueryParameterTypeReference({
                 };
             }
 
+            if (schema.value.value.schemas.length === 2) {
+                const [firstSchema, secondSchema] = schema.value.value.schemas;
+                if (
+                    firstSchema != null &&
+                    secondSchema != null &&
+                    hasSamePrimitiveValueType({ array: firstSchema, primitive: secondSchema })
+                ) {
+                    return {
+                        value: buildTypeReference({
+                            schema: Schema.optional({
+                                nameOverride: schema.nameOverride,
+                                generatedName: schema.generatedName,
+                                value: secondSchema,
+                                description: schema.description,
+                                groupName: undefined
+                            }),
+                            context,
+                            fileContainingReference
+                        }),
+                        allowMultiple: true
+                    };
+                }
+                if (
+                    firstSchema != null &&
+                    secondSchema != null &&
+                    hasSamePrimitiveValueType({ array: secondSchema, primitive: firstSchema })
+                ) {
+                    return {
+                        value: buildTypeReference({
+                            schema: Schema.optional({
+                                nameOverride: schema.nameOverride,
+                                generatedName: schema.generatedName,
+                                value: firstSchema,
+                                description: schema.description,
+                                groupName: undefined
+                            }),
+                            context,
+                            fileContainingReference
+                        }),
+                        allowMultiple: true
```

Comment:
should we be checking the [explode](https://github.com/fern-api/fern/blob/512b16a989ccd08a2cb0b1b0318a815bfa5aa13c/packages/cli/openapi-parser/src/__test__/fixtures/hume/openapi.yaml#L99) key in OAI ?

Reply:
Discussed offline - 

It looks like the current handling was primarily meant to handle comma-separated query parameters, so we should definitely gate this feature on the `explode` key (even if it requires editing OpenAPI schemas to specify this explicitly). Otherwise we would most likely break a variety of other use cases.

I'll follow-up with that change here before merging.

Reply:
On second thought, it looks like this is the default behavior for _any_ query parameter array (e.g. [reference](https://github.com/fern-api/fern/blob/512b16a989ccd08a2cb0b1b0318a815bfa5aa13c/packages/cli/openapi-ir-to-fern/src/buildQueryParameter.ts#L98)). Either way, we'll want to revisit this once we actually support different query parameter schemes (e.g. comma-separated, explode, etc).